libraries/stdlib/samples/test/samples/collections/arrays.kt

```diff
@@ -534,4 +534,54 @@ class Arrays {
 
     }
 
+    class Constructors {
+        @Sample
+        fun doubleArrayOf() {
+            val doubleArray = doubleArrayOf(1.0, 2.5, 3.14)
```

Comment:
It might make sense to showcase that these functions produce an empty array when invoked without any arguments. WDYT?